test/upgrade_SUITE.erl

```diff
@@ -52,48 +52,54 @@ do_find_make_cmd() ->
 
 %% Manipulate the release.
 
+do_copy(Example0) ->
+	Example = atom_to_list(Example0),
+	{ok, CWD} = file:get_cwd(),
+	do_exec_log("cp -r " ++ CWD ++ "/../../examples/" ++ Example ++ " " ++ CWD),
+	Dir = CWD ++ "/" ++ Example,
+	do_exec_log("sed -i s/\"include \\.\\.\\/\\.\\.\\/erlang.mk\"/\"include ..\\/..\\/..\\/erlang.mk\"/ " ++ Dir ++ "/Makefile"),
```

Comment:
Try changing `sed -i` into `sed -i.bak`?

Reply:
That seems to do the trick indeed :)